### PR TITLE
feat: add a fuzz test for kclvm-parser.parse_expr.

### PR DESCRIPTION
**Pull request by zong-zhe**
Note:
1. "PanicInfo.to_json_string" has been modified.
2. Some problems detected by the fuzz test will be solved soon.

What:
1. Add a fuzz test for kclvm-parser.parse_expr.
2. "PanicInfo.to_json_string" has been modified.
3. Remove some unusd imports.
4. In order to adapt to the new "PanicInfo.to_json_string",
   some code have been changed.

How:
1. Add "fuzz/fuzz_parser" in kclvm-parser for "parse_expr".
2. Remove some unusd imports.
3. "PanicInfo.to_json_string" has been modified by "serde_json::to_string".
4. Change all “$__kcl_PanicInfo__$” to “__kcl_PanicInfo__”.

Why:
1. Add a fuzz test in order to detect more bugs.
2. "serde" provides a better interface for serialization/deserialization of “PanicInfo”.
3. Serialization/Deserialization of “PanicInfo" is helpful for catching panic errors in fuzz.

- With pull requests:
  - Open your pull request against `main`
  - Your pull request should have no more than two commits, if not you should squash them.
  - It should pass all tests in the available continuous integration systems such as GitHub Actions.
  - You should add/modify tests to cover your proposed code changes.
  - If your pull request contains a new feature, please document it on the README.

### CODE REVIEW

File: kclvm/parser/fuzz/fuzz_targets/fuzz_parser.rs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+use serde_json;
+use std::panic::catch_unwind;
+use std::panic::set_hook;
+
+#[derive(Arbitrary, Debug)]
+enum ParserMethod<'a> {
+    ParseExpr { src: &'a str },
+}
+
+fuzz_target!(|method: ParserMethod| {
```
Comment (Peefy) [inline kclvm/parser/fuzz/fuzz_targets/fuzz_parser.rs:15]: You can put fuzz tests in the kclvm/tests folder, which is divided into parser fuzz tests.

Comment (Peefy) [reply kclvm/parser/fuzz/fuzz_targets/fuzz_parser.rs:15]: Add a fast access to fuzz test in makefile?

**Review by Peefy — APPROVED** [review]
LGTM

**Review by chai2010 — APPROVED** [review]
LGTM

**Review by chai2010 — CHANGES_REQUESTED** [review]
pls fix all "$__kcl_PanicInfo__$" in python: https://github.com/KusionStack/KCLVM/blob/dev/zong-zhe/add_kclvm_fuzzer/internal/kclvm_py/program/exec/kclvm_cli.py#L22

**Review by chai2010 — APPROVED** [review]

**Review by Peefy — APPROVED** [review]
LGTM

**Pull request merged**